include/ddc/kernels/splines/spline_evaluator.hpp

```diff
@@ -140,6 +232,25 @@ class SplineEvaluator
         return eval(coord_eval, spline_coef);
     }
 
+    /**
+     * @brief Evaluate spline function (described by its spline coefficients) on a mesh.
+     *
+     * The spline coefficients represent a spline function in a B-splines (basis splines). They can be obtained using a SplineBuilder.
+     *
+     * This is not a multidimensional evaluation. This is a batched 1D evaluation. It means for each coordinate of coords_eval, the evaluation is performed with the 1D set of spline coefficents of spline_coef identified with the same batch_domain_type::discrete_element_type which identifies the given coordinate of coords_eval (or the corresponding value of spline_eval which is computed).
```

Comment:
As discussed in #409 please break up this line.

> It means for each coordinate of coords_eval, the evaluation is performed with the 1D set of spline coefficents of spline_coef identified with the same batch_domain_type::discrete_element_type which identifies the given coordinate of coords_eval (or the corresponding value of spline_eval which is computed).

This sentence is huge and hard to follow. I'm going to try and make it easier to read but I would appreciate a second opinion

> This means that for each slice of coordinates identified by a batch_domain_type::discrete_element_type, the evaluation is performed with the 1D set of spline coefficients identified by the same batch_domain_type::discrete_element_type.

Reply:
A similar sentence appears below

Reply:
Yes I had some problems to make this part simple while giving the important idea to the reader.

Reply:
I agree your sentence is better